src/containers/editSettings/majorityVoting.tsx

```diff
@@ -48,27 +48,25 @@ export const EditMvSettings: React.FC<EditMvSettingsProps> = ({daoDetails}) => {
   const {isMobile} = useScreen();
 
   const {setValue, control} = useFormContext();
-  const {fields, replace} = useFieldArray({
-    name: 'daoLinks',
-    control,
-  });
-  const {errors, isValid, isDirty} = useFormState({
-    control,
-  });
+  const {fields, replace} = useFieldArray({name: 'daoLinks', control});
+  const {errors, isValid, isDirty} = useFormState({control});
 
-  const {data: daoToken, isLoading: tokensAreLoading} = useDaoToken(
-    daoDetails?.plugins?.[0]?.instanceAddress || ''
-  );
+  const pluginAddress = daoDetails?.plugins?.[0]?.instanceAddress as string;
+  const pluginType: PluginTypes = daoDetails?.plugins?.[0]?.id as PluginTypes;
+
+  const {data: daoToken, isLoading: tokensAreLoading} =
+    useDaoToken(pluginAddress);
 
   const {data: tokenSupply, isLoading: tokenSupplyIsLoading} = useTokenSupply(
-    daoToken?.address || ''
+    daoToken?.address ?? ''
   );
 
-  const {data, isLoading: settingsAreLoading} = usePluginSettings(
-    daoDetails?.plugins[0].instanceAddress as string,
-    daoDetails?.plugins[0].id as PluginTypes
-  );
-  const daoSettings = data as VotingSettings;
+  const {data: pluginSettings, isLoading: settingsAreLoading} =
+    useVotingSettings({
+      pluginAddress,
+      pluginType,
+    });
+  const votingSettings = pluginSettings as VotingSettings;
```

Comment:
At this point the `pluginSettings` object might be undefined, right? I'd specify it to avoid errors when trying to access some properties of the object
```suggestion
  const votingSettings = pluginSettings as VotingSettings | undefined;
```